changelog.py

```diff
@@ -245,6 +250,10 @@ def main():
     'ivadomed': {
         'labels': ['bug', 'dependencies', 'documentation', 'enhancement'],
         'generator': default_changelog_generator,
+    },
+    'data-multi-subject': {
```

Comment:
i would suggest to instead use `'labels': [None]` for the `'default'` case (otherwise for every new data repos, we will have to add the case here, and we have 20+ data repos)

Reply:
Ah, good point!